projects/systelab-login/src/lib/README.md

```diff
@@ -34,7 +34,8 @@ This is an example with the three components enabled.
                 [txtUsername]="txtUsername"
                 [txtRecoverProcessStarted]="txtRecoverProcessStarted"
                 [applicationName]="'App Name'" [moduleName]="'Module'" [version]="'1.4. (build 2012)'"
-                [isLoading]="isLoading">
+                [isLoading]="isLoading"
+                [noticeText}="'Notice Text'">
```

Comment:
@RubKrmon, the bracket needs to be changed in the README.md
` [noticeText]="'Notice Text'">`